codecs/mozjpeg_enc/example.html

```diff
@@ -27,6 +27,7 @@
       get_result_pointer: Module.cwrap('get_result_pointer', 'number', []),
       get_result_size: Module.cwrap('get_result_size', 'number', []),
     };
+    console.log('Version:', api.version().toString(16));
```

Comment:
Remove log. @surma is this from you?

Reply:
Yes, it’s just the example pages for the codecs to see that they are actually using the right version. This is on master already.